components/collaboration/InviteToCampaignModal.tsx

```
@@ -0,0 +1,216 @@
+};
+
+const InviteToCampaignModal: React.FC<Props> = ({
+  visible,
+  onClose,
+  collaborations,
+  onInvite,
+}) => {
+  const [selected, setSelected] = useState<string[]>([]);
+  const theme = useTheme();
+  const colors = Colors(theme);
+  const styles = useMemo(() => useStyles(colors), [colors]);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)